src/main/java/seedu/duke/common/Messages.java

```diff
@@ -146,28 +155,36 @@
     public static final String MESSAGE_QUIZ_DELETE_ERROR_NON_NUMBER = ":( OOPS!!! Please indicate in NUMERALS, "
             + "which quiz you'd like to delete!\n";
     public static final String MESSAGE_INVALID_HELP_COMMAND = ":( OOPS!!! Are you trying to take a quiz or add/delete a"
-            + " quiz question? Enter 'help' to check the correct format!\n";
+            + " quiz question? Enter 'help' to check the correct format!";
     public static final String MESSAGE_EMPTY_QUIZ_LIST = "Quiz list is empty. Add some!";
-    public static final String MESSAGE_QUIZ_INDEX_OUT_OF_BOUND = "There is not such a question in your quiz list!";
+    public static final String MESSAGE_QUIZ_INDEX_OUT_OF_BOUND = "There is no such question in your quiz list!";
+    public static final String MESSAGE_QUIZ_QUESTION_NOT_FOUND = "Question not found";
+    public static final String MESSAGE_QUIZ_ANSWER_NOT_FOUND = "Answer not found";
+    public static final String MESSAGE_QUIZ_OPTIONS_NOT_FOUND = "Options not provided";
+    public static final String MESSAGE_QUIZ_ADD_SUCCESSFUL = "Quiz question added!";
+    public static final String MESSAGE_QUIZ_INVALID_ANS_PROVIDED = "OOPS! Incorrect answer format! "
+            + "Your answer can only be either 1, 2, 3 or 4!";
+    public static final String MESSAGE_QUIZ_FULL_MARKS = "Congratulations! You get full marks in your last attempt!";
```

Comment:
Shld it be you have gotten full marks.. instead?

Reply:
I think "you got full marks" sounds better